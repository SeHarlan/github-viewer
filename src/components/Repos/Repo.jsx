import React from 'react';
import PropTypes from 'prop-types';

const Repo = ({ name }) => (
  <h3>{name}</h3>
);

Repo.propTypes = {
  name: PropTypes.string.isRequired
};
export default Repo;
