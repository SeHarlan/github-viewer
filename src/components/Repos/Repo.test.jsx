import React from 'react';
import { shallow } from 'enzyme';
import Repo from './Repo.jsx';

describe('Repo component', () => {
  it('renders Repo', () => {
    const wrapper = shallow(<Repo repoName="test repo" />);
    expect(wrapper).toMatchSnapshot();
  });
});
