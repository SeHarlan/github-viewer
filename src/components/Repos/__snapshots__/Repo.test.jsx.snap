// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repo component renders Repo 1`] = `
<h3>
  test repo
</h3>
`;
